content/en/docs/concepts/storage/volumes.md

```diff
@@ -605,6 +605,42 @@ spec:
       type: Directory
 ```
 
+{{< caution >}}
+It should be noted that the `FileOrCreate` mode does not create the parent directory of the file. If the parent directory of the mounted file does not exist, the pod fails to start. To ensure that this mode works, you can try to mount directories and files separately, as shown below
+{{< /caution >}}
+
+#### Example Pod(FileOrCreate)
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dnsutils
+  namespace: default
+spec:
+  containers:
+  - name: dnsutils
+    image: gcr.io/kubernetes-e2e-test-images/dnsutils:1.3
```

Comment:
Most of the other examples use `k8s.gcr.io/test-webserver` as image. Better follow that and also adapt the name and command.